.env.example

```
@@ -0,0 +1,74 @@
+# =============================================================================
+# Get these from https://www.reddit.com/prefs/apps
+REDDIT_CLIENT_ID=your_reddit_client_id
+REDDIT_CLIENT_SECRET=your_reddit_client_secret
+REDDIT_USER_AGENT=Trendit/1.0 by /u/yourusername
+
+# =============================================================================
+# AUTH0 CONFIGURATION
+# =============================================================================
+# Get these from your Auth0 Dashboard → Applications
+AUTH0_DOMAIN=your-domain.auth0.com
+AUTH0_CLIENT_ID=your_auth0_client_id
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The AUTH0_CLIENT_ID key should go before the AUTH0_DOMAIN key

(UnorderedKey)